powergenome/renewables_clusters.py

```diff
@@ -199,6 +203,9 @@ def build_clusters(
         max_clusters
             Maximum number of resource clusters to compute.
             If `None`, no clustering is performed; resources are returned unchanged.
+        max_lcoe
+            Select only the resources with a levelized cost of electricity (lcoe)
+            below this maximum. Takes precedence over `min_capacity`.
```

Comment:
Does it take precedence? Looks like `min_capacity` is still used, just with a pre-filtered dataframe.

Reply:
It does take precedence, in the sense that `max_lcoe` is enforced regardless of what `min_capacity` is set. Selected capacity will go below `min_capacity` if `max_lcoe` is low enough.